libmdbx/mdbx.h

```diff
@@ -738,7 +738,7 @@ enum MDBX_constants {
 /** \brief The name of the lock file in the environment */
 #define MDBX_LOCKNAME "/mdbx.lck"
 /** \brief The name of the data file in the environment */
-#define MDBX_DATANAME "/mdbx.dat"
+#define MDBX_DATANAME "mdbx.dat"
```

Comment:
Please revert this.
We can't change MDBX defs as they would get overwritten when MDBX code is updated